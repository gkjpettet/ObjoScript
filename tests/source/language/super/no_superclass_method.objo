class Base {}

class Derived is Base {
foo {
# Expect error: Base does not implement 'doesNotExist(_)'.
super.doesNotExist(1)
}
}

Derived().foo