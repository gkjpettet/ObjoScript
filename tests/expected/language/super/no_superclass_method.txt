The superclass (`Base`) of `Derived` does not define `doesNotExist(_)`.